docs/pages/database-access/faq.mdx

```diff
@@ -7,7 +7,12 @@ description: Frequently asked questions about Teleport Database Access
 
 ## Which database protocols does Teleport Database Access support?
 
-Teleport Database Access currently supports PostgreSQL and MySQL protocols.
+Teleport Database Access currently supports PostgreSQL and MySQL protocols, with
```

Comment:
If it's coming to 7.0 we should update previews and link here with a date, would be more helfpul

Reply:
Done.